packages/lime-system/files/usr/lib/lua/lime/generic_config.lua

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/lua
+
+local config = require("lime.config")
+local utils = require("lime.utils")
+
+gen_cfg = {}
+
+gen_cfg.NODE_ASSET_DIR = '/etc/lime-assets/node/'
+gen_cfg.COMMUNITY_ASSET_DIR = '/etc/lime-assets/community/'
+
+function gen_cfg.clean()
```

Comment:
empty function. please remove.

Reply:
` clean` and `configure` are mandatory for all lime modules, maybe I can improve the comments.

Reply:
thanks!